app/service.js

```diff
@@ -42,13 +42,17 @@ function getDocumentMetadata(url, window) {
     title: metadata.title,
     description: metadata.description,
     favicon_url: metadata.icon_url ? makeUrlAbsolute(url, metadata.icon_url) : makeUrlAbsolute(url, '/favicon.ico'),
-    images: [{
+    images: []
+  };
+
+  if (metadata.image_url) {
+    responseData.images = [{
```

Comment:
nit: Instead of overwriting the `responseData.images[]` array, we could just `Array.push()` to it here. 
Probably not a big deal either way since we're currently only ever returning a single image.